play/fraction-bingo/compare/app.js

```diff
@@ -0,0 +1,253 @@
+var diffLevel = "",
+    fract1 = "",
+    fract2 = "",
+    problem = "",
+    correctOp = "",
+    boardType = "",
+    width = 0,
+    ans = "",
+    table,
+    noSolBtn,
+    score = 0,
+    numSolved = 0;
+
+function startGame() {
+    // Get selected difficulty level                                                                                             
+    diffLevel = $("input[name='difficulty']:checked").val();
+
+    // If no difficulty level is selected, do not advance                                                                        
+    if (!diffLevel) {
+        return;
+    }
+
+    // Get selected board size                                                                                                   
+    if (document.getElementById("three").checked) {
+        document.getElementsByName("board3")[0].style.display = "";
+	boardType = "board3";
+	width = 3;
+    } else if (document.getElementById("four").checked) {
+        document.getElementsByName("board4")[0].style.display = "";
+	boardType = "board4";
+	width = 4;
+    } else if (document.getElementById("five").checked) {
+        document.getElementsByName("board5")[0].style.display = "";
+	boardType = "board5";
+	width = 5;
+    } else {
+        return;
+    }
+
+    generateProblem();
+    fillBoard();
+
+    document.getElementById("start").style.display = "none";
+    document.getElementById("game").style.display = "";
+}
+
+function generateProblem() {
+    fract1;
+    fract2;
+    if (diffLevel == "easy") {
+	fract1 = getRandomFraction(0, 9);
```

Comment:
I would advise replacing these numbers with constants that would be declared at the top of the file, something like:
```js
const EASY_MIN = 0; 
const EASY_MAX = 9;
```